hapi/vision/transforms/transforms.py

```diff
@@ -64,10 +66,13 @@ class Compose(object):
     def __init__(self, transforms):
         self.transforms = transforms
 
-    def __call__(self, data):
+    def __call__(self, *data):
```

Comment:
1. Add usage in Compose interface.
2. `__call__` interface is not exposed to doc.  But need to explain how to call in somewhere

Reply:
Done, thanks!